docs/sources/installation/configuration.md

```diff
@@ -650,6 +650,20 @@ Alert notifications can include images, but rendering many images at the same ti
 This limit will protect the server from render overloading and make sure notifications are sent out quickly. Default
 value is `5`.
 
+
+### execute_caculate_rule_timeout 
```

Comment:
This is not updated to reflect the new namings. Would be nice to include here what the default values are as well.